doc/api/timers.md

```diff
@@ -478,6 +451,8 @@ const interval = 100;
 [`setImmediate()`]: timers.md#timers_setimmediate_callback_args
 [`setInterval()`]: timers.md#timers_setinterval_callback_delay_args
 [`setTimeout()`]: timers.md#timers_settimeout_callback_delay_args
-[`util.promisify()`]: util.md#util_util_promisify_original
+[`timersPromises.setImmediate()`]: timers.md#timerspromisessetimmediatevalue-options
+[`timersPromises.setInterval()`]: timers.md#timerspromisessetintervaldelay-value-options
+[`timersPromises.setTimeout()`]: timers.md#timerspromisessettimeoutdelay-value-options
```

Comment:
```suggestion
[`timersPromises.setImmediate()`]: #timers_timerspromises_setimmediate_value_options
[`timersPromises.setInterval()`]: #timers_timerspromises_setinterval_delay_value_options
[`timersPromises.setTimeout()`]: #timers_timerspromises_settimeout_delay_value_options
```

Reply:
Thank you for pointing this out. I've made the changes. It's a little confusing that the markdown header anchor links are different from the ones in the generated HTML docs on nodejs.org. Is this documented anywhere?

Would be nice if all the anchor links the markdown docs in the repo also worked and/or were automatically prefixed when the HTML docs are generated. I'm assuming there's probably some reason this doesn't happen automatically.

Reply:
> It's a little confusing that the markdown header anchor links are different from the ones in the generated HTML docs on nodejs.org. Is this documented anywhere?

I don't think it's documented, it's been this way since at least 2012 (https://github.com/nodejs/node/commit/217bb47b0271df61dc93a37a440404794679245a#diff-55254552ad384f461c2d30801ccf911060301b22751baeea57dba5916f3d24e6R142-R146), I don't know if GitHub had markdown header anchor at that time.
FWIW when I'm working on the docs, to get the correct link reference, I use `make docserve -j` to open the generated docs on my browser, and I copy the links straight from the TOC. Not ideal but it does the work.

> Would be nice if all the anchor links the markdown docs in the repo also worked and/or were automatically prefixed/converted when the HTML docs are generated. I'm assuming there's probably some reason this doesn't happen automatically.

I reckon that'd be really nice to have that kind of tool, I think the reason it hasn't happened (yet?) is because no one has put the work to make it happen. The real upside of a tool like that would be to be able to browse the docs from GitHub UI and locally from a text editor with decent Markdown support! If you (or anyone else) feel that's a challenge you want work on, you are more than welcome 😊

Reply:
Apparently there is a syntax for this: https://www.markdownguide.org/extended-syntax/#heading-ids
I don't know if our tooling already supports it, but that could help a lot to have the heading IDs hard coded in the Markdown files.

Reply:
Neat! Thank you for the explanation and tips!